universe/Conditions.cpp

```diff
@@ -9933,6 +9933,18 @@ unsigned int ValueTest::GetCheckSum() const {
 std::unique_ptr<Condition> ValueTest::Clone() const
 { return std::make_unique<ValueTest>(*this); }
 
+std::array<const ValueRef::ValueRef<double>*, 3> ValueTest::ValuesDouble() const
```

Comment:
I think these could be defined in the header where declared

Reply:
Moved here.